src/components/Video/PreviewControls/PreviewControls.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useEffect } from 'react';
+import styles from './PreviewControls.module.scss';
+import isTouchDevice from '../../../utils/isTouchDevice';
+import useMousePosition from '../../../utils/hooks/useMousePosition';
+
+const PreviewControls = ({ handleClick }: { handleClick: () => void }) => {
+  const isMobile = isTouchDevice();
```

Comment:
Touch device !== mobile, I assume we're wanting to actually check for touch here

Reply:
Yea you're right. It's supposed to be checking for touch